dash/testing/browser.py

```diff
@@ -65,8 +66,11 @@ def __init__(
         self._percy_run = percy_run
         self._pause = pause
 
-        self._driver = until(self.get_webdriver, timeout=1)
-        self._driver.implicitly_wait(2)
+        if driver:
+            self._driver = driver
+        else:
+            self._driver = until(self.get_webdriver, timeout=1)
+            self._driver.implicitly_wait(2)
```

Comment:
Do we want the `implicitly_wait` inside here or would it be better to always call that even if you provide your own `driver`?

Reply:
Lets set the `implicitly_wait(2)` for all, since there is no easy way to confirm if the driver already has one set.